airbyte-integrations/connectors/destination-elasticsearch-strict-encrypt/build.gradle

```diff
@@ -40,14 +40,3 @@ dependencies {
 
     integrationTestJavaImplementation project(':airbyte-connector-test-harnesses:acceptance-test-harness')
 }
-
-repositories {
```

Comment:
❓ am I understanding that if a connector needs to still define a repository in their own `build.gradle` they still can? (just thinking about optionality in the future)

Reply:
Indeed no they can't, not while 
```
    repositoriesMode = RepositoriesMode.FAIL_ON_PROJECT_REPOS
```
is set. Other settings allow subprojects to either inherit the repo list from the root and extend it, or to replace it.

It's fine to add more entries to the global list and gate them with a whitelist of artifact group IDs. This shouldn't happen too often. If I'm wrong, we certainly should revisit this.

Reply:
Sounds fair to me!